TransportationSplitter.py

```diff
@@ -376,56 +497,130 @@ def add_lr_split_points(split_points, lrs, segment_id, original_segment_geometry
     # Get the length of the projected segment
     geod = pyproj.Geod(ellps="WGS84")
     line_length = geod.geometry_length(original_segment_geometry)
```

Comment:
Switch this to use `get_length` as well?

Reply:
good catch! length is actually passed on as it's already calculated by the caller, i missed this